app/navigation/Routes.tsx

```diff
@@ -0,0 +1,35 @@
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import {NavigationContainer} from '@react-navigation/native';
+import React from 'react';
+import {TabButton} from '../components';
+import {TabScreens} from '../constants';
+import {applicationStyle} from '../theme';
+
+const Routes = () => {
+  const Tab = createBottomTabNavigator();
```

Comment:
missing type

Reply:
Okay, but make sure to do the next PR otherwise I reject all next PR without warning.

Reply:
okay , will keep in mind